Extension/src/Debugger/extension.ts

```diff
@@ -130,6 +131,12 @@ export function dispose(): void {
     disposables.forEach(d => d.dispose());
 }
 
+function sshTerminal(node: TargetLeafNode): void {
+    const terminal: vscode.Terminal = vscode.window.createTerminal(`SSH: ${node.name}`);
+    terminal.sendText(`ssh ${node.name}`);
```

Comment:
To consider: Can 'name' contain any characters that we'd need to quote or escape?

Reply:
Strings are escaped automatically.

Reply:
Did some tests
- Quotes in names are not supported at all in SSH config, so nothing will show up in our view.
- Everything after spaces in names are omitted

So we only need to add quotes around node.name to make the corner cases work.